src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -99,6 +101,7 @@ export interface NetEmitterOptions {
     csharpGeneratorPath: string;
     "clear-output-folder"?: boolean;
     "save-inputs"?: boolean;
+    "model-namespace"?: boolean;
```

Comment:
This configuration property name sounds more like an input of the model name space. Maybe the following is better.
```suggestion
    "separate-model-namespace"?: boolean;
```

Reply:
It is from Michael's doc, we may update that after further discussing with all the names.  For now, I suggest leave it as it is.

Reply:
Yes. it is better to change to `enable-model-namespace` or `separate-model-namespace`. but now this flag has already used in mgmt, and we need to reuse the existing one now. Maybe we will change it to more clear name when we refact the whole project both mgmt and data-plane.